molecule_vagrant/modules/vagrant.py

```diff
@@ -147,216 +145,72 @@
-
-      # Custom
-      provider['options'].each { |key, value|
-        eval("vmware.#{key} = #{value}")
-      }
-
-      # Raw Configuration
-      if provider['raw_config_args']
-        provider['raw_config_args'].each { |raw_config_arg|
-          eval("vmware.#{raw_config_arg}")
-        }
-      end
-
-      if provider['override_args']
-        provider['override_args'].each { |override_arg|
-          eval("override.#{override_arg}")
-        }
-      end
-    end
-  end
-
-  ##
-  # Parallels
-  ##
-
-  if provider['name'] == 'parallels'
-    config.vm.provider provider['name'] do |parallels, override|
-      parallels.memory = provider_memory
-      parallels.cpus = provider_cpus
-
-      # Custom
-      provider['options'].each { |key, value|
-        eval("parallels.#{key} = #{value}")
-      }
-
-      # Raw Configuration
-      if provider['raw_config_args']
-        provider['raw_config_args'].each { |raw_config_arg|
-          eval("parallels.#{raw_config_arg}")
-        }
-      end
-
-      if provider['override_args']
-        provider['override_args'].each { |override_arg|
-          eval("override.#{override_arg}")
-        }
-      end
-    end
-  end
-
-  ##
-  # Libvirt
-  ##
-
-  if provider['name'] == 'libvirt'
-    config.vm.provider provider['name'] do |libvirt, override|
-      libvirt.memory = provider_memory
-      libvirt.cpus = provider_cpus
-
-      # Custom
-      provider['options'].each { |key, value|
-        eval("libvirt.#{key} = #{value}")
-      }
-
-      # Raw Configuration
-      if provider['raw_config_args']
-        provider['raw_config_args'].each { |raw_config_arg|
-          eval("libvirt.#{raw_config_arg}")
-        }
-      end
-
-      if provider['override_args']
-        provider['override_args'].each { |override_arg|
-          eval("override.#{override_arg}")
-        }
-      end
-    end
-  end
-
-
-  ##
-  # Instances
-  ##
-
-  if vagrant_config_hash['instance']
-    instance = vagrant_config_hash['instance']
-    config.vm.define instance['name'] do |c|
-      c.vm.hostname = instance['name']
-
-      if instance['interfaces']
-        instance['interfaces'].each { |interface|
-          c.vm.network "#{interface['network_name']}",
-                       Hash[interface.select{|k| k != 'network_name'}.map{|k,v| [k.to_sym, v]}]
-        }
-      end
-
-      if instance['raw_config_args']
-        instance['raw_config_args'].each { |raw_config_arg|
-          eval("c.#{raw_config_arg}")
-        }
-      end
+  {% for instance in instances %}
+  config.vm.define "{{ instance.name }}" do |c|
+    # Box options
+    c.vm.box = "{{ instance.box }}"
+    {{ 'c.vm.box_version = "{}"'.format(instance.box_version) if instance.box_version }}
+    {{ 'c.vm.box_url = "{}"'.format(instance.box_url) if instance.box_url }}
+
+    c.vm.hostname = "{{ instance.name }}"
+    c.vm.synced_folder ".", "/vagrant", disabled: {{ ruby_format(not instance.synced_folder) }}
```

Comment:
If someone wants to change the synced folder, how can it be done ?

Reply:
They would have to set it in `instance_raw_config_args`, e.g.:
```yml
instance_raw_config_args:
  - 'vm.synced_folder "foo/", "/bar/"
```